pkg/k8s/rule_translate.go

```diff
@@ -63,8 +66,16 @@ func (k RuleTranslator) TranslateEgress(r *api.EgressRule) error {
 
 func (k RuleTranslator) populateEgress(r *api.EgressRule) error {
 	for _, service := range r.ToServices {
-		// TODO: match services by labels
-		if service.K8sService == api.K8sServiceNamespace(k.Service) {
+		matchesSelector := false
```

Comment:
Put the "matching" logic of the service in a single version to avoid code repetition here and on line 90-99.